### PR TITLE
fix spacing

### PR DESCRIPTION
**Pull request by jreis13**
Why:
* Spacing was not matching the design.

How:
* Adjusted spacing to match design

### CODE REVIEW

File: frontend/src/components/Forms/FormLayout/index.module.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .headerWrapper {
-  margin-bottom: 48px;
+  margin-bottom: 16px;
```
Comment (jreis13) [inline frontend/src/components/Forms/FormLayout/index.module.css:2]: this is in relation to a new line of text will be added in another PR

**Review by joaojusto — COMMENTED** [review]
So this is the reference: 
Screenshot 2022-08-22 at 11 19 52

This is the running version on this PR:
Screen Shot 2022-08-22 at 11 21 09

It seems that font sizes are completely off as well as the input sizes.

**Review by joaojusto — APPROVED** [review]
